src/gwt/src/org/rstudio/studio/client/workbench/prefs/views/AppearancePreferencesPane.R

```diff
@@ -3,9 +3,11 @@ fivenum <- function(x) {
   
   # handle empty input
   n <- length(x)
-  if (n == 0)
+  if (n == 0) {
+     message("No input")
```

Comment:
I wonder if there's a more realistic example we could use, since a typical R function like this wouldn't show messages.

Maybe we could add something like:

```
if (!is.numeric(x)) {
  stop("'x' must be a numeric vector")
}
```

to validate that `x` is a numeric vector?